network-store-client/src/main/java/com/powsybl/network/store/client/PreloadingNetworkStoreClient.java

```diff
@@ -871,4 +912,16 @@ public void removeConfiguredBuses(UUID networkUuid, int variantNum, List<String>
         ensureCached(ResourceType.CONFIGURED_BUS, networkUuid, variantNum);
         delegate.removeConfiguredBuses(networkUuid, variantNum, configuredBusesId);
     }
+
+    @Override
+    public Optional<ExtensionAttributes> getExtensionAttributes(UUID networkUuid, int variantNum, ResourceType resourceType, String identifiableId, String extensionName) {
+        ensureExtensionAttributesCached(resourceType, networkUuid, variantNum, extensionName);
+        return delegate.getExtensionAttributes(networkUuid, variantNum, resourceType, identifiableId, extensionName);
+    }
+
+    @Override
+    public Map<String, ExtensionAttributes> getAllExtensionsAttributesByIdentifiableId(UUID uuid, int workingVariantNum, ResourceType resourceType, String id) {
```

Comment:
rename to getIdentifiableExtensions

Reply:
See discussion https://github.com/powsybl/powsybl-network-store/pull/411#discussion_r1701590040